lib/src/models/avatar_options.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:acter_avatar/acter_avatar.dart';
+
+/// Helper class for defining avatar type.
+/// Currently supported modes are Space/Group Chat (default), DM, Group DM.
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'avatar_options.freezed.dart';
+
+@freezed
+abstract class AvatarOptions with _$AvatarOptions {
+  /// Default usage .i.e. Space/Group Chat. Group avatars will appear as secondary badges.
```

Comment:
I think enum values could be reduced to have one default constructor which can have mode of Space/GroupChat avatar with its optional badges params. The other types can be initialised via their named constructors.